client_internal.go

```diff
@@ -231,6 +231,9 @@ func (c *Client) loadDefaultPaymailConfig() (err error) {
 		)
 	}
 
+	paymailLogger := c.Logger().With().Str("subservice", "go-paymail").Logger()
+	c.options.paymail.serverConfig.options = append(c.options.paymail.serverConfig.options, server.WithLogger(&paymailLogger))
```

Comment:
There is an error here 
`undefined: server.WithLogger`. Is it ok ?

Reply:
I will be ok when the paymail PR will be merged, new version will come and then after update we will have this method

Reply:
Now it works